package.json

```diff
@@ -22,9 +22,12 @@
   "scripts": {
     "build": "tsc --build tsconfig.json",
     "lint": "polkadot-dev-run-lint",
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
+    "generate:acala": "npx parachain-launch generate --config=configs/acala.yml --y",
+    "generate:karura": "npx parachain-launch generate --config=configs/karura.yml --y"
   },
   "dependencies": {
+    "@open-web3/parachain-launch": "^1.2.0",
```

Comment:
why depends on self?

Reply:
This was so i could run it with `npx` but you're right, should have just run the `parachain-launch.js` locally. In any case this is removed now given the yarn scripts have been removed.